cms/settings/base.py

```diff
@@ -305,3 +305,5 @@
 NHSEI_MAX_MENU_CAPTION_LENGTH = 70
 # TODO - turn this on when we go live, it breaks imports all ove the place
 WAGTAILREDIRECTS_AUTO_CREATE = False
+# pagination page size for search results
+RESULTS_PER_PAGE = 10
```

Comment:
Having improved it beyond how I'd left it, I'm sorry -- now it's moved, is it worth being explicitly `SEARCH_RESULTS_PER_PAGE`?